### PR TITLE
Add dependencies to readme

### PR DESCRIPTION
**Pull request by nick-enoent**
Fix typo in maestro comment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ configuration across the remaining aggregators in a group
 In this current release, Maestro does not start and start __ldmsd__
 daemons, however, this feature is planned for the future.
 
+## Dependencies
+python-hostlist
+etcd3
```
Comment (tom95858) [inline README.md:25]: Is this the actual package name?

File: README.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ configuration across the remaining aggregators in a group
 In this current release, Maestro does not start and start __ldmsd__
 daemons, however, this feature is planned for the future.
 
+## Dependencies
+python-hostlist
```
Comment (tom95858) [inline README.md:24]: I think I failed to include this file, I will do so.

File: README.md (hunk 1 above)
Comment (nick-enoent) [reply README.md:25]: Yes, these are the actual package names as installed with Python3 pip (pip3).

Comment (tom95858) [reply README.md:25]: @nick-enoent, can you remove the python-hostlist dependency from the README? Also add a source for the etcd3 code.

Comment (nick-enoent) [reply README.md:25]: Sure thing

**Pull request merged**